src/index.tsx

```diff
@@ -48,6 +49,7 @@ const problemOrdinal = urlParams.problem || DefaultProblemOrdinal;
 const {investigation, problem} = unit.getProblem(problemOrdinal) ||
                                  unit.getProblem(DefaultProblemOrdinal);
 const showDemoCreator = urlParams.demo;
+const createStores = urlParams.dataflow !== undefined ? createDataflowStores : createBaseStores;
```

Comment:
So adding the `dataflow` URL parameter allows us to toggle between the Control Panel and a `DocumentWorkspaceComponent` and adding a URL parameter with `unit=dataflow` brings us to a `DocumentWorkspaceComponent` with modified left panels and tile toolbar.  In practice, how will these fit together (the `dataflow` and `unit` URL params)?

Reply:
In the short term both are required to get the full effect. Ultimately, the authoring of the unit should support all of the desired variation, at which point the URL parameter can go away.

Reply:
Got it.  I was using one at a time.  Not sure what impact that will have on my other comments.